Exercise_4.py

```diff
@@ -7,19 +17,53 @@ def __init__(self, data):
         self.right = None
           
 """ Inorder traversal of a binary tree"""
-def inorder(temp): 
-  
-    
-  
-  
+def inorder(temp): #(left,root,right)
```

Comment:
Great work implementing iterative traversal. Try recursive as well as it is easier and shorter.